homeassistant/components/media_player/plex.py

```diff
@@ -45,24 +43,24 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
     def update_devices():
         """ Updates the devices objects. """
         try:
-            devices = plexuser.devices()
+            devices = plexserver.clients()
         except BadRequest:
             _LOGGER.exception("Error listing plex devices")
             return
 
         new_plex_clients = []
         for device in devices:
-            if (all(x not in ['client', 'player'] for x in device.provides)
-                    or 'PlexAPI' == device.product):
+            # For now, let's allow all deviceClass types
+            if device.deviceClass in []:
                 continue
```

Comment:
This is more or less equivalent to:

    if None:
        continue

Which can never happen. Please just remove those likes and the comment.